yarn.lock

```diff
@@ -9,24 +9,24 @@
   dependencies:
     "@babel/highlight" "^7.12.13"
 
-"@babel/helper-validator-identifier@^7.12.11":
```

Comment:
Since you got rid of the `@ethersproject/abstract-signer` import we should be able to get rid of this diff.